pipeline/transform/matches_test.py

```diff
@@ -4,13 +4,15 @@
     convert_matches_from_df,
     filter_recent_matches,
 )
-from pipeline.types import Match
+from pipeline.types import Match, MatchMetadata
 
 
 def test_convert_matches_from_df():
     df = pd.DataFrame([{"users": {"A", "B"}, "extraData": "ignore"}])
     actual = convert_matches_from_df.run(df)
-    expected = [Match(users={"A", "B"})]
+    expected = [
+        Match(users={"A", "B"}, metadata=MatchMetadata(generator="blank"))
+    ]
```

Comment:
If you remove what we added, the test fails because the match metadata's generator field equals something related to magic mock.
I did not fully understand the problem.

<img width="1070" alt="Screenshot 2022-07-23 at 12 26 55 PM" src="https://user-images.githubusercontent.com/47644298/180616149-ec9716ea-e4f2-4a74-971b-d90283542b13.png">

Reply:
Yeah, this is a weird error. There's no mock in this test at all. At least it's working now :)